data/recommendations.html

```diff
@@ -111,14 +119,20 @@ <h2>ClamAV and ClamTK</h2>
                                 light-weight, on-demand scanner for Linux systems.&zwnj;</p>
                         </div>
                         <div class="text-md-right">
-                            <span id="clamav-remove" class="alert alert-success"> <i
-                                    class="material-icons">&#xE876;</i>&zwnj;ClamAV is installed&zwnj;</span><a
-                                id="clamav-install" class="btn btn-success hvr-shadow"
-                                onclick="cmd('install?clamav');"> &nbsp;<span
-                                    class="material-icons">&#xE2C0;</span>&zwnj;Install & Configure
-                                ClamAV&zwnj;&nbsp;</a>
-                            <span id="clamav-status" class="hidden"><img src="img/welcome/processing.gif"
-                                    width="24px" height="24px" /> &zwnj;Applying Changes...&zwnj;</span>
+                            <div id="clamav-remove">
+                                <span id="clamav-remove" class="alert alert-success"> 
+                                    <i class="material-icons">&#xE876;</i>&zwnj;ClamAV is installed&zwnj;</span>
+                                    <a class="btn btn-danger" onclick="cmd('remove?clamav');">
+                                        <span class="material-icons">&#xE872;</span><span
+                                        class="button-text">&zwnj;Remove&zwnj;</span>
+                                    </a>
+                                    <span id="clamav-status" class="hidden"><img src="img/welcome/processing.gif"
```

Comment:
recommendations was always a little odd - it has been a "one click" install - but cannot remove via this screen.  This is unlike budgie extras.

We need to either remove the "remove button" here to be consistent with the other installs ... or fix the whole page to have remove buttons for everything.  

The "sometimes remove" and "others no remove" is inconsistent.